joomla-projects/privacy-framework
# Fix terms and privacy article links

Pull Request for Issue # .

### Summary of Changes

Same as 2nd test in https://github.com/joomla/joomla-cms/pull/20412

### Testing Instructions

On single language site select terms and privacy articles in `plg_user_terms` and `plg_system_privacyconsent`. Go to registration page and check article links.

### Expected result
Correct links, e.g. `https://example.com/article-url?tmpl=component`

### Actual result
Links with `&lang=*` appended `https://example.com/article-url?tmpl=component&lang=*`


### Documentation Changes Required
No
